lib/util/include/hse_util/logging.h

```diff
@@ -146,6 +162,8 @@ hse_log_deregister(int code);
 bool
 hse_log_push(struct hse_log_fmt_state *state, bool indexed, const char *name, const char *value);
 
+/* Structured logging support.
```

Comment:
Where you have doccomments, use `/**` imo.

Reply:
This has no markup, so not really a "doccomment"?  I intended it to be a "section heading" for the src file.